list_updatable_packages

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIRECTORIES=( $(grep --files-with-matches '^# template: foreman_plugin' packages/*/*/*.spec | xargs --max-args=1 dirname) )
```

Comment:
For now this is just `foreman_plugin` as a safe start but there's little reason to stick to it. We could easily also add others:
```suggestion
DIRECTORIES=( $(grep --files-with-matches '^# template: (foreman|smart_proxy|hammer)_plugin' packages/*/*/*.spec | xargs --max-args=1 dirname) )
```